dlc-manager/src/contract/ord_descriptor.rs

```diff
@@ -0,0 +1,375 @@
+//! Description of contracts using an ordinal as part of the collateral.
+
+use bitcoin::Transaction;
+use dlc::{
+    ord::{OrdPayout, SatPoint},
+    RangePayout,
+};
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+use crate::error::Error;
+
+use super::{enum_descriptor::EnumDescriptor, numerical_descriptor::NumericalDescriptor};
+
+/// Descriptor for a contract involving an ordinal sat being traded.
+#[derive(Clone, Debug)]
+#[cfg_attr(
+    feature = "serde",
+    derive(Serialize, Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct OrdDescriptor {
+    /// The descriptor for the contract outcomes.
+    pub outcome_descriptor: OrdOutcomeDescriptor,
+    /// The original location of the ordinal within the UTXO set.
+    pub ordinal_sat_point: SatPoint,
```

Comment:
`SatPoint` is the name used in the `ord` implementation: https://github.com/ordinals/ord/blob/master/src/sat_point.rs#L4

`output_index` seems like the index of the output within the transaction, but it's more than that since it also provides the offset for the actual satoshi.

Reply:
Ok, if that's how ORD does it, let's stick to the structure!